pkg/apis/servicecatalog/types.go

```diff
@@ -443,6 +451,14 @@ type PlanReference struct {
 	// ClusterServicePlanRef below.
 	ClusterServicePlanExternalName string
 
+	// ClusterServiceClassExternalID is the broker's external id for the class.
+	//
+	// Immutable.
+	ClusterServiceClassExternalID string
+
+	// ClusterServicePlanExternalID is the broker's external id for the plan.
```

Comment:
Add that this is immutable? also in `v1beta1`

Reply:
It's not immutable though, right? Just like ClusterServicePlanExternalName and ClusterServicePlan, you can upgrade the plan later depending on the class.

Reply:
iirc you can upgrade the plan later, so I'd agree

Reply:
You're right